File: js/5347.js
const fs = require('fs');

const inputs = fs.readFileSync('/dev/stdin').toString().split('\n');

const n = +inputs[0];
const answer = [];
for (let i = 1; i <= n; i++) {
  const [a, b] = inputs[i].split(' ').map(Number);
  answer.push(lcm(a, b));
}

console.log(answer.join('\n'));

function gcd(a, b) {
  return a % b === 0 ? b : gcd(b, a % b);
}

function lcm(a, b) {
  return (a * b) / gcd(a, b);
}

File: c++/5347.cpp
#include <iostream>

using namespace std;

long long gcd(long long a, long long b) {
  if (a % b == 0)
    return b;
  else
    return gcd(b, a % b);
}

long long lcm(long long a, long long b) { return a * b / gcd(a, b); }

int main() {
  // freopen("example.txt", "r", stdin);

  int n;
  cin >> n;
  for (int i = 0; i < n; i++) {
    int a, b;
    cin >> a >> b;
    cout << lcm(a, b) << '\n';
  }

  return 0;
}
